invenio_circulation/transitions/transitions.py

```diff
@@ -70,8 +69,8 @@ def _ensure_valid_extension(loan):
     extension_count += 1
 
     if extension_count > extension_max_count:
-        msg = 'Max extension count reached `{0}`'.format(extension_max_count)
-        raise TransitionConstraintsViolation(msg=msg)
+        msg = "Max extension count reached '{0}'".format(extension_max_count)
+        raise TransitionConstraintsViolation(description=msg)
```

Comment:
Also here?

Reply:
I have created different Exceptions for these as suggested.